src/user/dto/create-user.dto.ts

```diff
@@ -0,0 +1,45 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Gender, UiTheme } from '@prisma/client';
+import { IsEnum, IsString } from 'class-validator';
+
+export class CreateUserDto {
+  @ApiProperty({ example: 'John', description: 'First Name' })
+  @IsString()
+  firstName: string;
+
+  @ApiProperty({ example: 'Doe', description: 'Last Name' })
+  @IsString()
+  lastName: string;
+
+  @ApiProperty({ example: '+359 888 888 888', description: 'Phone Number' })
+  @IsString()
+  phoneNumber: string;
+
+  @ApiProperty({ example: 'https://example.com/photo.png', description: 'Photo URL' })
+  @IsString()
+  photoUrl: string;
+
+  @ApiProperty({ example: 'MALE', description: 'Male' })
+  @IsEnum(Gender)
+  gender: Gender;
+
+  @ApiProperty({ example: 'Bulgaria', description: 'Country' })
+  @IsString()
+  country: string;
+
+  @ApiProperty({ example: 'en', description: 'Language' })
```

Comment:
Looks like we need inventory table for available countries to validate county on FE and BE. @bakhromov96  @hgoziev  what do you think?